tests/unit/testsuite/Bolt/Boltpay/Block/Checkout/BoltpayTest.php

```diff
@@ -25,9 +42,26 @@ public function setUp()
             ->getMock()
         ;
 
+        Mage::unregister('_helper/boltpay');
+        $this->helperMock = $this->getMockBuilder('Bolt_Boltpay_Helper_Data')
+            ->setMethods(
+                ['notifyException', 'logException']
```

Comment:
we use "array()" in the project. It's better to follow one style.

Reply:
Actually, in our code base, we have been using both the old 5.3 and below syntax, `array()` and the 5.4 and above syntax of `[]` for a long time.  One example is in the Product page checkout code that you are familiar with.

https://github.com/BoltApp/bolt-magento1/blob/master/app/code/community/Bolt/Boltpay/Block/Catalog/Product/Boltpay.php#L67-L81

Our coding standards are enforced by the official Magento 1 MEQP1 rule set at severity Level 10.  This allows us to be officially allowed in the Magento 1 Marketplace.  It does not restrict the use of short array syntax.

https://github.com/magento/marketplace-eqp

At any rate, we can unify the syntax in a separate PR if it is for some reason deemed important.